Editor/BFImporter.cs

```diff
@@ -88,7 +89,9 @@ public static void DoImportBitmapFont(string fntPatn)
             
             AssetDatabase.SaveAssets();
 
-#if UNITY_5_5_OR_NEWER
+#if UNITY_2019_2_OR_NEWER
+            // No-op
+#elif UNITY_5_5_OR_NEWER
```

Comment:
Please change to UNITY_2018_4_OR_NEWER.
It works fine in unity 2018

Reply:
Hey, thanks for the suggestion.

So just to be clear, do you suggest changing the `#if` on line 92 or the `#elif` on line 94?

By saying 'it works fine in Unity 2018' do you mean that the `ReloadFont()` call is not needed in Unity 2018.4, so the code becomes:

```suggestion
#if UNITY_2018_4_OR_NEWER
            // No-op
#elif UNITY_5_5_OR_NEWER
```
or that the `ReloadFont()` call is needed for Unity 2018.4, so the code becomes:

```suggestion
#if UNITY_2019_2_OR_NEWER
            // No-op
#elif UNITY_2018_4_OR_NEWER
```

Reply:
```suggestion
#if UNITY_2018_4_OR_NEWER
            // No-op
#elif UNITY_5_5_OR_NEWER
```
It works fine in unity version 2018.4 without `ReloadFont`

Reply:
Done!